wdfn-server/waterdata/templates/macros/components.html

```diff
@@ -11,21 +11,6 @@ <h2 class="usa-acordion__heading">
             </h2>
             <div id="iv-data-table-container" class="usa-accordion__content usa-prose"></div>
         </div>
-        <div class="usa-alert usa-alert--info provisional-data-alert" role="alert" hidden>
```

Comment:
I think I would like to see the markup for the provisional tag here rather than in the javascript. Could it be moved here? Anything that decreases the amount of javascript needed is good!

Reply:
Great thought!! done.